src/Module/Admin/Storage.php

```diff
@@ -145,6 +146,7 @@ protected function content(array $request = []): string
 			'$save_reload'           => DI::l10n()->t('Save & Reload'),
 			'$noconfig'              => DI::l10n()->t('This backend doesn\'t have custom settings'),
 			'$form_security_token'   => self::getFormSecurityToken("admin_storage"),
+			'$storagebackend_ro_txt' => !DI::config()->isWritable('storage', 'name') ? DI::l10n()->t('Changing the current backend is prohibited because it is set by an environment variable') : '',
```

Comment:
You need to do this for the `$is_writable` variable to be set in the templates.
```suggestion
			'$storagebackend_ro_txt' => !DI::config()->isWritable('storage', 'name') ? DI::l10n()->t('Changing the current backend is prohibited because it is set by an environment variable') : '',
			'$is_writable' => DI::config()->isWritable('storage', 'name'),
```